### ISSUE TITLE
On launch of ANY MO (using newest one atm Beta 1) Failed to create modlist.txt 

### ISSUE THREAD

**Issue by xirangeix** [TanninOne/modorganizer#724]
On launch of ANY MO (using newest one atm Beta 1) Failed to create modlist.txt . Not sure why. Please help:"( it says C drive but Fo4 AND MO are both on my F drive. HMM.

Really want to use the MO master race for F04 Too:(

Please and thanks.

**Comment by alexjet85** [comment]
delete : "C:\Users\"username"\AppData\Local\ModOrganizer and launch MO again

**Comment by TanninOne** [comment]
Is this resolved then? @alexjet85 do you know why the dir has to be deleted? Is this a permission issue?

**Comment by xirangeix** [comment]
Yeah I figured it out myself. Deleting that does work. Still all the directories were wonky. Had to make manual folders in another MO folder for mods ect, then set them in MO program.

**Comment by alexjet85** [comment]
@TanninOne i have seen this when installing over an old MO setup, it would seem to me like a permission issue indeed.

(Sorry for the bad english)
